### PR TITLE
Shaheer/diel flow change

### PR DESCRIPTION
**Pull request by shaheer-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/appstore/src/components/cfds-listing/cfds-listing.scss [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
         flex-direction: column;
         @include desktop {
-            overflow: scroll;
+            overflow: auto;
         }
     }
     &__scrollable-content {
         @include desktop {
-            overflow: scroll;
+            overflow: auto;
```
Comment (shaheer-deriv) [inline packages/appstore/src/components/cfds-listing/cfds-listing.scss:11]: An intermittent issue of displaying scrollbar area in the jurisdiction modal while connecting laptop to monitor was encountered earlier by harvinraj. Today i encountered the same issue and fixing it here.

File: packages/cfd/src/Containers/cfd-password-modal.tsx [hunk 2]
```diff
@@ -841,24 +856,28 @@ const CFDPasswordModal = ({
             getDxCompanies()[category as keyof TDxCompanies][type as keyof TDxCompanies['demo' | 'real']].short_title;
         const jurisdiction_label =
             jurisdiction_selected_shortcode && getFormattedJurisdictionCode(jurisdiction_selected_shortcode);
-        const mt5_platform_label = jurisdiction_selected_shortcode !== Jurisdiction.MALTA_INVEST ? 'MT5' : '';
+        const mt5_platform_label =
+            is_eu_user || jurisdiction_selected_shortcode !== Jurisdiction.MALTA_INVEST ? 'MT5' : '';
```
Comment (amina-deriv) [inline packages/cfd/src/Containers/cfd-password-modal.tsx:860]: we dont need this.
For eu users,jurisdiction_selected_shortcode will be malatainvest. lets refactor this

File: packages/cfd/src/Containers/cfd-password-modal.tsx [hunk 3]
```diff
@@ -847,42 +862,27 @@ const CFDPasswordModal = ({
...
-                    platformName = mt5_platform_label;
-                    break;
-                case CFD_PLATFORMS.DERIVEZ:
-                    platformName = 'Deriv Ez';
-                    break;
-                default:
-                    platformName = 'Deriv X';
-                    break;
-            }
-
```
Comment (shaheer-deriv) [inline packages/cfd/src/Containers/cfd-password-modal.tsx:865]: `platformName` is not used anywhere.

File: packages/cfd/src/Containers/cfd-password-modal.tsx [hunk 4]
```diff
@@ -961,7 +980,11 @@ const CFDPasswordModal = ({
                 is_open={should_show_success}
                 toggleModal={closeModal}
                 onCancel={closeModal}
-                onSubmit={platform === CFD_PLATFORMS.MT5 && !is_selected_mt5_verified ? closeModal : closeOpenSuccess}
+                onSubmit={
+                    !is_eu_user && platform === CFD_PLATFORMS.MT5 && !is_selected_mt5_verified
+                        ? closeModal
+                        : closeOpenSuccess
```
[the remark this answers is not in the document]
Comment (shaheer-deriv) [reply packages/cfd/src/Containers/cfd-password-modal.tsx:996]: @amina-deriv in this case too, code flows to `closeOpenSuccess` method, inside which the modal will be closed.
apart from the logic written here, there is also some logic inside `closeOpenSuccess` function.
Please let me know if this is not the flow expected.

File: packages/cfd/src/Containers/cfd-password-modal.tsx [hunk 5]
```diff
@@ -827,12 +835,19 @@ const CFDPasswordModal = ({
...
+                return is_eu_user || is_selected_mt5_verified ? localize('Transfer now') : localize('OK');
             }
             return localize('Transfer now');
         }
         return localize('Continue');
-    }, [platform, account_type, is_selected_mt5_verified]);
+    }, [platform, account_type, is_eu_user, is_selected_mt5_verified]);
+
+    const success_modal_cancel_label = React.useMemo(() => {
+        if (is_eu_user && account_type.category === 'real' && platform === CFD_PLATFORMS.MT5) {
```
Comment (amina-deriv) [inline packages/cfd/src/Containers/cfd-password-modal.tsx:846]: wont this effect non-eu flow? can you double check the flow?

File: packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx [hunk 6]
```diff
@@ -86,9 +86,7 @@ const ContractInfo = ({
     const setHintMessage = () => {
         if (['VANILLALONGCALL', 'VANILLALONGPUT'].includes(type)) {
             return (
-                <Localize
-                    i18n_default_text='The payout at expiry is equal to the payout per point multiplied by the difference between the final price and the strike price.'
-                />
+                <Localize i18n_default_text='The payout at expiry is equal to the payout per point multiplied by the difference between the final price and the strike price.' />
```
Comment (amina-deriv) [inline packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx:89]: why do we have changes in trader package?

File: packages/cfd/src/Containers/cfd-password-modal.tsx [hunk 7]
```diff
@@ -847,42 +862,27 @@ const CFDPasswordModal = ({
...
+                            platform: platform === CFD_PLATFORMS.MT5 ? 'MT5' : getCFDPlatformLabel(platform),
                             category: category_label,
-                            jurisdiction_selected_shortcode:
-                                platform === CFD_PLATFORMS.MT5 && !show_eu_related_content ? jurisdiction_label : '',
+                            jurisdiction_selected_shortcode: platform === CFD_PLATFORMS.MT5 ? jurisdiction_label : '',
                         }}
                         components={[<span key={0} />, <strong key={1} className='cfd-account__platform' />]}
                     />
-                    {platform === CFD_PLATFORMS.DXTRADE ? (
+                    {is_eu_user || platform === CFD_PLATFORMS.DXTRADE ? (
```
Comment (amina-deriv) [inline packages/cfd/src/Containers/cfd-password-modal.tsx:885]: lets not add MT5 conditions with Derivx.
You can add condition to ReviewMessageForMT5 in line- 209
 if (is_selected_mt5_verified) {
        return (
            <Localize i18n_default_text='To start trading, top-up funds from your Deriv account into this account.' />
        );

File: packages/cfd/src/Containers/cfd-password-modal.tsx [hunk 8]
```diff
@@ -847,42 +862,27 @@ const CFDPasswordModal = ({
...
-                default:
-                    platformName = 'Deriv X';
-                    break;
-            }
-
             return (
                 <React.Fragment>
                     <Localize
-                        i18n_default_text='Congratulations, you have successfully created your {{category}} <0>{{platform}}</0> <1>{{type}} {{jurisdiction_selected_shortcode}}</1> account. '
+                        i18n_default_text='Congratulations, you have successfully created your {{category}} <0>{{deriv_keyword}} {{platform}}</0> <1>{{type}} {{jurisdiction_selected_shortcode}}</1> account. '
```
Comment (amina-deriv) [inline packages/cfd/src/Containers/cfd-password-modal.tsx:870]: can you double check the message?
As per design , it should be real Deriv MT5 CFD'S Malta account. 
we need to add the keyword CFD's

File: packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx (hunk 6 above)
Comment (shaheer-deriv) [reply packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx]: someone force pushed.

File: packages/cfd/src/Containers/cfd-password-modal.tsx (hunk 8 above)
Comment (shaheer-deriv) [reply packages/cfd/src/Containers/cfd-password-modal.tsx:870]: Yes, this is the modal currently being displayed
Screenshot 2023-06-26 at 12 29 20 PM

**Review by suisin-deriv — APPROVED** [review]

**Pull request merged**